packages/cli/lib/validators/marketplaceValidators/theme/SectionValidator.js

```diff
@@ -0,0 +1,116 @@
+const {
+  ANNOTATION_KEYS,
+  getAnnotationValue,
+  getFileAnnotations,
+} = require('@hubspot/cli-lib/templates');
+const BaseValidator = require('../BaseValidator');
+const { VALIDATOR_KEYS } = require('../../constants');
+
+const SECTION_LIMIT = 50;
+
+class SectionValidator extends BaseValidator {
+  constructor(options) {
+    super(options);
+
+    this.errors = {
+      LIMIT_EXCEEDED: {
+        key: 'limitExceeded',
+        getCopy: ({ limit, total }) =>
+          `Section limit exceeded. Themes can only have ${limit} sections, but this theme has ${total}`,
+      },
+      MISSING_LABEL: {
+        key: 'missingLabel',
+        getCopy: ({ filePath }) =>
+          `Missing required property for ${filePath}. The section is missing the "label" property`,
+      },
+      MISSING_SCREENSHOT_PATH: {
+        key: 'missingScreenshotPath',
+        getCopy: ({ filePath }) =>
+          `Missing required property for ${filePath}. The section is missing the "screenshotPath" property`,
+      },
+      MISSING_DESCRIPTION: {
+        key: 'missingDescription',
+        getCopy: ({ filePath }) =>
+          `Missing required property for ${filePath}. The section is missing the "description" property`,
+      },
+    };
+  }
+
+  // Validates:
+  // - All sections have a 'description' annotation
```

Comment:
NIT: quotes

Reply:
Thanks, resolved in https://github.com/HubSpot/hubspot-cli/pull/546/commits/c95e30d9d7a7e260f4d0af1b8c2efc6a77be658e